demo/roles/users/tasks/main.yml

```diff
@@ -1,6 +1,14 @@
+- name: Create hpccloud user
+  become: yes
+  become_user: root
+  user: name=hpccloud shell=/bin/bash uid=1002
+
+- name: Give hpccloud passwordless sudo
+  become: yes
+  become_user: root
+  lineinfile: "dest=/etc/sudoers state=present regexp='^hpccloud' line='hpccloud ALL=(ALL) NOPASSWD: ALL'"
+
```

Comment:
Why do we need this here? Doesn't the other users role create the hpccloud user?